packages/dd-trace/src/config.js

```diff
@@ -806,31 +542,289 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
+    this._setBoolean(opts, 'startupLogs', options.startupLogs)
     this._setTags(opts, 'tags', tags)
+    this._setBoolean(opts, 'telemetry.logCollection', options.iastOptions &&
+    (options.iastOptions === true || options.iastOptions.enabled === true))
+    this._setBoolean(opts, 'traceId128BitGenerationEnabled', options.traceId128BitGenerationEnabled)
+    this._setBoolean(opts, 'traceId128BitLoggingEnabled', options.traceId128BitLoggingEnabled)
+    this._setString(opts, 'version', options.version || tags.version)
+  }
+
+  _isCiVisibility () {
+    return coalesce(
+      this.options.isCiVisibility,
+      this._defaults['isCiVisibility']
+    )
+  }
+
+  _isCiVisibilityItrEnabled () {
+    return coalesce(
+      process.env.DD_CIVISIBILITY_ITR_ENABLED,
+      true
+    )
+  }
+
+  _getHostname () {
+    const DD_CIVISIBILITY_AGENTLESS_URL = process.env.DD_CIVISIBILITY_AGENTLESS_URL
+    const url = DD_CIVISIBILITY_AGENTLESS_URL ? new URL(DD_CIVISIBILITY_AGENTLESS_URL)
+      : getAgentUrl(this._getTraceAgentUrl(), this.options)
+    const DD_AGENT_HOST = coalesce(
+      this.options.hostname,
+      process.env.DD_AGENT_HOST,
+      process.env.DD_TRACE_AGENT_HOSTNAME,
+      '127.0.0.1'
+    )
+    return DD_AGENT_HOST || (url && url.hostname)
+  }
+
+  _getSpanComputePeerService () {
+    const DD_TRACE_SPAN_ATTRIBUTE_SCHEMA = validateNamingVersion(
+      coalesce(
+        this.options.spanAttributeSchema,
+        process.env.DD_TRACE_SPAN_ATTRIBUTE_SCHEMA
+      )
+    )
+
+    const peerServiceSet = (
+      this.options.hasOwnProperty('spanComputePeerService') ||
+      process.env.hasOwnProperty('DD_TRACE_PEER_SERVICE_DEFAULTS_ENABLED')
+    )
+    const peerServiceValue = coalesce(
+      this.options.spanComputePeerService,
+      process.env.DD_TRACE_PEER_SERVICE_DEFAULTS_ENABLED
+    )
+
+    const spanComputePeerService = (
+      DD_TRACE_SPAN_ATTRIBUTE_SCHEMA === 'v0'
+        // In v0, peer service is computed only if it is explicitly set to true
+        ? peerServiceSet && isTrue(peerServiceValue)
+        // In >v0, peer service is false only if it is explicitly set to false
+        : (peerServiceSet ? !isFalse(peerServiceValue) : true)
+    )
+
+    return spanComputePeerService
+  }
+
+  _isCiVisibilityGitUploadEnabled () {
+    return coalesce(
+      process.env.DD_CIVISIBILITY_GIT_UPLOAD_ENABLED,
+      true
+    )
+  }
+
+  _isCiVisibilityManualApiEnabled () {
+    return coalesce(
+      process.env.DD_CIVISIBILITY_MANUAL_API_ENABLED,
+      false
+    )
+  }
+
+  _isTraceStatsComputationEnabled () {
+    return coalesce(
+      this.options.stats,
+      process.env.DD_TRACE_STATS_COMPUTATION_ENABLED,
+      getIsGCPFunction() || getIsAzureFunctionConsumptionPlan()
+    )
+  }
+
+  _getTraceAgentUrl () {
+    return coalesce(
+      this.options.url,
+      process.env.DD_TRACE_AGENT_URL,
+      process.env.DD_TRACE_URL,
+      null
+    )
+  }
+
+  // currently does not support dynamic/remote config
```

Comment:
Is this still true? It looks like it should be supported.

Reply:
yeah it is, just removed it, forgot to push